app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/common/SurveyToolbar.kt

```diff
@@ -0,0 +1,57 @@
+package com.kks.nimblesurveyjetpackcompose.ui.presentation.common
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.kks.nimblesurveyjetpackcompose.R
+
+@Composable
+fun SurveyToolbar(
+    modifier: Modifier = Modifier,
+    showBack: Boolean,
+    showClose: Boolean,
+    onClickClose: () -> Unit,
```

Comment:
`onCloseClick` would be more meaningful, IMHO.
```suggestion
    onCloseClick: () -> Unit,
```

Reply:
IMO, `onClickClose` is the shorthand of `onClickCloseButton`.  Grammarly, I think it is in format of `Subject` + `Verb` + `Object`.